Runtime/Scripts/EdgeEventsManager.cs

```diff
@@ -273,7 +273,7 @@ public void StartEdgeEvents(MobiledgeXIntegration mxi)
     IEnumerator StartEdgeEventsLocation(EdgeEventsConnection connection)
     {
       yield return StartCoroutine(UpdateLocation());
-      connection.PostLocationUpdate(location).ConfigureAwait(false).GetAwaiter().GetResult();
+      connection.PostLocationUpdate(location).ConfigureAwait(false);
```

Comment:
Makes sense, yes. This background operation longer conflicts with DeviceInfo being on the UI thread, right?

Reply:
Yes